src/validation.js

```diff
@@ -37,7 +37,7 @@ function validateQueryAndPathParameters(operation, operationName, args, options)
 
     // there should be no missing required arguments
     Object.keys(parameters).forEach((name) => {
-        if (parameters[name] && !(name in queryAndPathArgs)) {
+        if (parameters[name] && !(queryAndPathArgs[name])) {
```

Comment:
maybe `args.get(name)` - it will allow us to support body in the future

Reply:
hmm what about 0?

Reply:
This is just for query/path. Body will be handled later.